terraform/infrastructure/output.tf

```diff
@@ -57,3 +57,7 @@ output "firehose" {
 output "mi" {
   value = module.mi
 }
+
+output "permissions_bucket" {
```

Comment:
Again ive called this permissions bucket but we should probably adjust that to be consistent to authorization_store